.docker/test-OpenData.ini

```diff
@@ -31,13 +28,9 @@ cache_dir = /tmp/%(ckan.site_id)s/
 beaker.session.key = ckan
 beaker.session.type = file
 
-# This is the secret token that the beaker library uses to hash the cookie sent
-# to the client. `paster make-config` generates a unique value for this each
-# time it generates a config file.
+SECRET_KEY = bSmgPpaxg2M+ZRes3u1TXwIcE
 beaker.session.secret = bSmgPpaxg2M+ZRes3u1TXwIcE
```

Comment:
could you add comment that this param is deprecated and goes away post 2.11+

Reply:
Okay, done.